File: 2021/0026_removeDuplicates/0805/removeDuplicates.ts
function removeDuplicates(nums: number[]): number {
    let slow = 0;
    let len = nums.length;
    for (let fast = 0; fast < len; fast++) {
        if (nums[fast] == nums[slow]) { continue }
        slow += 1;
        nums[slow] = nums[fast];
    }
    return slow + 1;
}

File: 2021/0026_removeDuplicates/0805/removeDuplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        slow = 0
        for fast in range(len(nums)):
            if nums[fast] == nums[slow]:
                continue
            slow += 1
            nums[slow] = nums[fast]
        return slow + 1 if nums else 0
